core/src/main/java/org/mobilitydata/gtfsvalidator/notice/Notice.java

```diff
@@ -93,4 +105,36 @@ public int hashCode() {
   public boolean isError() {
     return getSeverityLevel().ordinal() >= SeverityLevel.ERROR.ordinal();
   }
+
+  /** JSON exclusion strategy for notice context. It skips {@link Notice#severityLevel}. */
+  private static class NoticeExclusionStrategy implements ExclusionStrategy {
+    public boolean shouldSkipClass(Class<?> clazz) {
+      return false;
+    }
+
+    public boolean shouldSkipField(FieldAttributes f) {
```

Comment:
Why should we skip severityLevel?

Reply:
It was not a part of the context originally. When we export notices, we group them by (code, severityLevel) and then export contexts for each group.


```
    "notices": [
        {
            "code": "decreasing_or_equal_shape_distance",
            "severity": "ERROR",
            "totalNotices": 2517999,
            "notices": [
                {
                    "shapeId": "112321",
                    "csvRowNumber": 232516,
                    "shapeDistTraveled": 955.3,
                    "shapePtSequence": 10002,
                    "prevCsvRowNumber": 232515,
                    "prevShapeDistTraveled": 955.3,
                    "prevShapePtSequence": 10001
                },
                {
                    "shapeId": "112321",
                    "csvRowNumber": 232517,
                    "shapeDistTraveled": 955.3,
                    "shapePtSequence": 10003,
                    "prevCsvRowNumber": 232516,
                    "prevShapeDistTraveled": 955.3,
                    "prevShapePtSequence": 10002
                },
```